### ISSUE TITLE
ios PWA Showing navigation bar on chat selection 

### ISSUE THREAD

**Issue by davecrab** [danny-avila/LibreChat#2360]
### What happened?

On iOS when using it as a PWA it launches ok and is full screen. However when selecting a chat from the sidebar the top and bottom bars are showing as if it’s opened the chat in a new tab. 
This was on a fresh install of librechat using the default docker compose
I’ve tried clearing the browser cache but had no luck. 

### Steps to Reproduce

1. Add librechat to the iOS home screen
2. Open side bar
3. Click on a previous chat

### What browsers are you seeing the problem on?

Mobile (iOS)

### Screenshots

![Screenshot 2024-04-08 at 8 23 13 PM jpeg](https://github.com/danny-avila/LibreChat/assets/65996799/803b0368-58d4-4f68-9233-4f482666cef8)
![Screenshot 2024-04-08 at 8 23 31 PM jpeg](https://github.com/danny-avila/LibreChat/assets/65996799/bdaeddb9-2b42-45a0-b303-2c9081486e1f)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by davecrab** [comment]
Hmm weird. Maybe it's something to do with iOS 17.5 or I just completely missed something. 

Added vite-plugin-pwa to it and built locally which seems to have fixed it for me. 

Thanks for taking a look

**Comment by danny-avila** [comment]
> Hmm weird. Maybe it's something to do with iOS 17.5 or I just completely missed something.
> 
> Added vite-plugin-pwa to it and built locally which seems to have fixed it for me.
> 
> Thanks for taking a look

interesting, if you think it's useful to the repo as a whole please make a PR!

**Comment by webzit** [comment]
It would be very helpful if @davecrab could contribute his fixes as a PR. I am experiencing the same issues on both iPadOS 17.4.1 and iOS 17.4.1.

**Comment by davecrab** [comment]
I have a pr in for it already 👍

**Comment by webzit** [comment]
Thank you, @davecrab,  I tested your PR and it works fine! 

I additional commented out "textAreaRef.current?.focus();" in  ./client/src/hooks/Input/useTextarea.ts to not always trigger the iOS Keyboard when switching the chat, as it is quite annoying in the pwa app.

I also tried to disable the Safari gestures (Previous/Next) in the pwa app, but to no avail. It would be nice to replace these gestures with expanding/hiding the side menus.
